### PR TITLE
Bootstrap update for prom filter stats

### PR DESCRIPTION
**Pull request by bianpengyuan**
Cherry-pick several commits to make master consistent with 1.3

### CODE REVIEW

File: tools/packaging/common/envoy_bootstrap_v2.json
Comment (howardjohn) [inline tools/packaging/common/envoy_bootstrap_v2.json]: these regexs are pretty complex. Should we have a test for them? The existing regex ones are worse

Comment (bianpengyuan) [reply tools/packaging/common/envoy_bootstrap_v2.json]: For sure we will need an integration test, but I'd rather not to do it in this PR. I create a issue https://github.com/istio/istio/issues/16722 to track this and will work on it after this PR merged.

**Review by mandarjog — APPROVED** [review]
/lgtm

This is a forward merge from 1.3

**Comment by mandarjog** [comment]
@duderino @geeknoid looks like I lost the ability to approve PRs for bootstrap. For telemetry we use a lot of bootstrap.

**Review by howardjohn — APPROVED** [review]

**Comment by bianpengyuan** [comment]
@istio/wg-user-experience-maintainers Can I get an approval for istioctl change? Two test golden files were updated. Thanks!

**Comment by bianpengyuan** [comment]
@ayj Could you help to approve this change? It has two small changes to istioctl tests. Thanks!

**Review by ayj — APPROVED** [review]

**Pull request merged**
